### PR TITLE
Release information beautification

### PR DESCRIPTION
**Pull request by StephanStS**
@fn753; @fpetru; @MichaIng; @Joulinar; @ravenclaw900:  
I generated a draft with some minor text changes in the release notes to start a discussion with you. 
- Q1: I generated a kind of "logo" for the upcoming release information. My idea is to share such a logo also on twitter, facebook, blogs, etc. What do you think about this?  
  See `Version #3`, possibly the logo is a bit too large...
  image

- Q2: Alternatively we could focus the next release within an admonitions block.  
  See `Version #2`

- Q3: I played with **bold** emphasizing to point out keywords within the admonition blocks of the new feature list:

  image
  If this is o.k., I would add some more emphasizing in bold.

### CODE REVIEW

File: docs/releases.md [hunk 1]
```diff
@@ -18,29 +18,46 @@ Welcome to **April 2021 release** :octicons-heart-16: of **DietPi**. It's an inc
...
+
+***Version #1:***  
+We already scheduled the next update:  
+***DietPi 7.2 will be launched on 2021-05-29***.
+
+***Version #2:***  
+
+!!! check "Next DietPi release"
+
+    As a new thing, we already scheduled the next update: ***DietPi 7.2 will be launched on 2021-05-29***.
```
Comment (Joulinar) [inline docs/releases.md:49]: Just have something like this in mind: "As a new feature, we will introduce shorter release cycle and a fixed schedule. The next update is already planned: ***DietPi 7.2 will be launched on 2021-05-29***. "

Comment (MichaIng) [reply docs/releases.md:49]: Agreed, that reads great. Only: `shorter release cycles` (plural)

Comment (StephanStS) [reply docs/releases.md]: @Joulinar, @MichaIng: Added your remarks. 

What about the three versions?

Comment (Joulinar) [reply docs/releases.md]: Personally I'm in favour of option 2. Probably we could introduce an image/logo later on. So we could have some more time looking into a layout and design?

Comment (fpetru) [reply docs/releases.md]: I also agree with option 2. Regarding the logo / image let's add them together in 7.2

**Comment by fpetru** [comment]
@StephanStS - To not open a new PR, could you please add to DietPi-Config, the latest change coming with 7.1.2 ?

_DietPi-Config | Worked around an issue in RPi, where selecting a sound card ended with an error prompt in certain circumstances. Many thanks to @sturbs for reporting this issue: #4306_

**Comment by MichaIng** [comment]
I love the idea with the logo. I'd just use capital letters for the first words:
> Next DietPi release:
> - Version: v7.2
> - Date: 2021-05-29

Making key words bold is also a good idea, especially in the admonitions blocks there the script/area is on a different level/layout already (compared to regular logs).

**Comment by fpetru** [comment]
In my view, the data of the next release (in May) should not be overemphasis. The scope of providing the release notes for 7.1 is to highlight the launched features and invite users to try them. Underlining too much that v7.2 comes soon, removes the incentive to migrate.  

Still, it's great to mention it and I like a lot option 2. It communicates both the new approach and also the future day of release.

**Comment by StephanStS** [comment]
Yes, I will do that.

-----Original Message-----

**Comment by Joulinar** [comment]
Fully agree on @fpetru

File: docs/releases.md (hunk 1 above)
Comment (StephanStS) [reply docs/releases.md]: Ok, we start with version 2.

**Comment by StephanStS** [comment]
> I love the idea with the logo. I'd just use capital letters for the first words:
> 
> > Next DietPi release:
> > 
> > * Version: v7.2
> > * Date: 2021-05-29
> 
> Making key words bold is also a good idea, especially in the admonitions blocks there the script/area is on a different level/layout already (compared to regular logs).

Changed within the PR.

**Comment by fpetru** [comment]
On small comment on the format of the next launch date 
> The next update is already planned: ***DietPi 7.2 will be launched on 2021-05-29***.

Please update it to 29 May 2021. Especially for US users, it becomes easier to read. Nothing wrong related to the international format yyyy-mm-dd, but it is not so commonly used for the launches.

**Comment by MichaIng** [comment]
> Please update it to 29 May 2021.

I like `yyyy-mm-dd`, as it's the only format the can be practically used everywhere, especially also for filenames and lists, correctly chronologically sortable etc and the only numerical format where the order of day, month and year is undoubtedly clear to everyone, compared to XX/YY/ZZ where sometimes days are in the front, sometimes in the middle, sometimes at the end (whether it was even used correctly or not) 😄.

**Comment by fpetru** [comment]
> > Please update it to 29 May 2021.
> 
> I like `yyyy-mm-dd`, as it's the only format the can be practically used everywhere, especially also for filenames and lists, correctly chronologically sortable etc and the only numerical format where the order of day, month and year is undoubtedly clear to everyone, compared to XX/YY/ZZ where sometimes days are in the front, sometimes in the middle, sometimes at the end (whether it was even used correctly or not) 😄.

ok

**Review by fpetru — APPROVED** [review]

**Comment by MichaIng** [comment]
Note on my last commit: The removed Chromium and Kodi "Changes" entries were duplicates that belong and were already below "Bug fixes".

**Review by MichaIng — APPROVED** [review]

**Pull request merged**